src/docs/currencies/bitcoin/introduction.mdx

```diff
@@ -2,11 +2,15 @@
 title: Introduction
 ---
 
-
 ## What is Bitcoin?
+
 Bitcoin was the first decentralized digital currency based on blockchain technology. It was created in 2008 by a mysterious and anonymous developer known as Satoshi Nakamoto, who released the white paper in a cryptography mailing list and later open-sourced the software that implements the protocol.
 
-### Supported Bitcoin Address Types
+### AirGap Bitcoin Wallet
+
+AirGap Bitcoin(BTC) Wallet is a convenient way to store and transfer BTC while storing your private key offline.
```

Comment:
What is this? I find the term "AirGap Bitcoin Wallet" confusing, I would expect this to be an app that I can download from the store.